### DISCUSSION TITLE
go here

### DISCUSSION THREAD

**Discussion by jxah** [jxah/solid-happiness#2] [General]
first things first make a python script called Q and A.py(the end).#1

**Comment by jxah** [comment]
even to you @escambron .

**Comment by maparce** [comment]
¿Estaría bien si pongo _ underscores en vez de espacios en el nombre del file?

**Reply by maparce** [reply]
Puedes subir el Python script que ya tienes hecho.

**Reply by jxah** [reply]
hmmmmmmmmmmmmmm

no

**Reply by jxah** [reply]
esto es locura
